src/main/java/org/aksw/gerbil/evaluate/ExtendedMacros.java

```diff
@@ -0,0 +1,39 @@
+package org.aksw.gerbil.evaluate;
+
+import org.aksw.gerbil.matching.EvaluationCounts;
+
+public class ExtendedMacros {
```

Comment:
The meaning of the class name is not clear to me. "Macro" is a very general term. Maybe you can come up with a better name?